### PR TITLE
Adding Leslie Sweet, More prezs

### PR DESCRIPTION
**Pull request by christoball**
adding Leslie Sweet's headshot and bio, updating front page events section, also adding other presentations

### CODE REVIEW

File: content/markdown/news/2019-09-18-forum-keynote-justen-noakes-heb-director-emergency-preparedness.md [hunk 1]
```diff
@@ -3,14 +3,35 @@ layout: news-article.njk
...
+
+As director of Public Affairs for H-E-B in Central Texas, Leslie Sweet is responsible for coordinating the company's community and public relations efforts in the region, with statewide strategic programming responsibilities.  
+
+Leslie is passionate about food education and understanding the farming process from the roots on up. She has traveled from produce fields in California to coffee fields in Panama, expanding her agricultural knowledge and experience. She served on the advisory board of the Texas Organic Agricultural Industry where she assisted the Texas Department of Agriculture in assessing the state of the organic agricultural industry.
+
+In 2012 Leslie helped launch a personal labor of love with H-E-B’s statewide military appreciation campaign.  This campaign honors the service and sacrifice of military service members and their families in Texas.  Since the launch, H-E-B has recognized and honored over 4,5000 H-E-B employees who are veterans, built 27 homes for severely wounded heroes, and continues to give strategic support to help military families where they need it most.
```
Comment (jwhaney) [inline content/markdown/news/2019-09-18-forum-keynote-justen-noakes-heb-director-emergency-preparedness.md]: fix 4,5000

**Comment by christoball** [comment]
@jwhaney fixed

**Comment by jwhaney** [comment]
@christoball, i think the rest looks good but i do have a suggestion. the headshot on the front page of tnris.org was changed to leslie's pic, but the front page of the 2019 forum events page was not changed. think we should change it there as well?

**Comment by christoball** [comment]
@jwhaney good suggestion, done.

**Review by jwhaney — APPROVED** [review]

**Pull request merged**